### PR TITLE
clean up mixed tab and spaces

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Long Dai <long0dai@foxmail.com>

# Description

Covert tab to 2 spaces in all all files mixed tab and spaces.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by artursouza — CHANGES_REQUESTED** [review]
Do not change auto-generated code.

**Comment by daixiang0** [comment]
@artursouza updated.

File: docker/Dockerfile-windows-php-base [hunk 1]
```diff
@@ -11,10 +11,10 @@ EXPOSE 80
 ENTRYPOINT ["C:\\ServiceMonitor.exe", "w3svc"]
 
 RUN powershell -Command \
-	[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
-	Invoke-WebRequest -Method Get -Uri https://windows.php.net/downloads/releases/latest/php-8.0-nts-Win32-vs16-x64-latest.zip -OutFile c:\php.zip ; \
-	Expand-Archive -Path c:\php.zip -DestinationPath c:\php ; \
-	Remove-Item c:\php.zip -Force
+  [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
```
Comment (mukundansundar) [inline docker/Dockerfile-windows-php-base:14]: nitpick: can we have the same amount of space in lines 6 and 7 as well?

File: tests/test-infra/clean_up.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@ echo "Trying to delete namespace..."
 kubectl delete namespace $1 --timeout=10m
 
 for pod in `kubectl get pods -n $1 -o name`; do
-	kubectl delete --force -n $1 $pod
+  kubectl delete --force -n $1 $pod
```
Comment (mukundansundar) [inline tests/test-infra/clean_up.sh:23]: nitpick: can we have the same amount of space in line 14 as well?

**Review by mukundansundar — CHANGES_REQUESTED** [review]

File: tests/test-infra/clean_up.sh (hunk 2 above)
Comment (daixiang0) [reply tests/test-infra/clean_up.sh:23]: done.

**Review by mukundansundar — APPROVED** [review]
lgtm

**Review by mukundansundar — CHANGES_REQUESTED** [review]
Do not change autogenerated code.

**Comment by daixiang0** [comment]
@mukundansundar seems codes back by mistake, sorry.

**Review by artursouza — APPROVED** [review]

**Review by mukundansundar — APPROVED** [review]

**Pull request merged**